app/core/models.py

```diff
@@ -289,3 +292,39 @@ def save(self, *args, **kwargs):
                 'Max of 1 active highlight fields has been reached.')
 
         return super(CourseInformationMapping, self).save(*args, **kwargs)
+
+
+class ReceiverEmailConfiguration(models.Model):
+    """Model for Email Configuration """
+
+    email_address = models.EmailField(
+        max_length=254,
+        help_text='Enter email personas addresses to send log data',
+        unique=True)
+
+    def get_absolute_url(self):
+        """ URL for displaying individual model records."""
+        return reverse('Configuration-detail', args=[str(self.id)])
+
+    def __str__(self):
+        """String for representing the Model object."""
+        return f'{self.id}'
+
+    def save(self, *args, **kwargs):
+        email_verification(self.email_address)
+        return super(ReceiverEmailConfiguration, self).save(*args, **kwargs)
+
+
+class SenderEmailConfiguration(models.Model):
+    """Model for Email Configuration """
+
+    sender_email_address = models.EmailField(
+        max_length=254,
+        help_text='Enter sender email address to send log data from',
+        default='openlxphost@gmail.com')
```

Comment:
We should use an example email address that is 100% definitely not a real email address -- you never know with Gmail.